scorecards.go

```diff
@@ -0,0 +1,122 @@
+package opslevel
+
+import (
+	"fmt"
+)
+
+type ScorecardId struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+}
+
+type Scorecard struct {
+	ScorecardId
+
+	Description   string      `graphql:"description"` // optional
+	Filter        Filter      `graphql:"filter"`      // optional
```

Comment:
This should be a pointer with `omitempty` because it is causing CLI to show phantom filters when there aren't any.

```
    "Filter": {
        "Connective": "",
        "HtmlURL": "",
        "Id": null,
        "Name": "",
        "Predicates": null
    },
```

Can do the same for `Description`

Reply:
Historically - i don't use pointers on non Input types so things are initialized.  For dependent resources like this usually we do a `.Id != nil` check if you want to know if the object is set or not.

The `omitempty` won't leave out the string pointers - its a JSON struct tag so its only used for marshaling a struct to JSON not the other way around.

I'm open to change this though if you think it doesn't make sense or would be better being fully nil if its not in the response payload.

Reply:
That makes sense to me. Changing this would be outside of the PR's scope, so let's leave it as-is.